vscode_client_2/README.md

```diff
@@ -2,28 +2,31 @@
 
 Mainframe messenger is an easy way how you can send messages from Visual Studio Code to another user on TSO on green screen. The user you send messages to must be connected to a specific mainframe connection what you configure in extension settings. Mainframe messenger has Zowe conformance and uses Zowe profile to connect to mainframe.
 
-## How to use Mainframe Messenger
-
+## Prerequisites
 1. Make sure you have Java version 11 on your computer
 2. Make sure you have a Zowe zosmf profile with credentials defined
-3. Open Visual Studio Code and install the Mainframe messenger extension
-4. In extension settings, write the name of your Zowe profile under "zowe profile"
-5. Write your mainframe username and password in "username" and "password"
-6. If you want use a custom name on your messages, write the name under "custom name"
-7. Under "host" write the URL of your mainframe connection in format http(s)://host(port)
-8. Create a new file in your VS Code workspace with the extension .txt
-9. Write your message in the file 
+ 
+## Install Mainframe Messenger
+1. Open Visual Studio Code and install the Mainframe messenger extension
+
+## How to use Mainframe Messenger
+1. In extension settings, write the name of your Zowe profile under "zowe profile"
+2. Write your mainframe username and password in "username" and "password"
+3. If you want use a custom name on your messages, write the name under "custom name"
+4. Under "host" write the URL of your mainframe connection in format http(s)://host(port)
```

Comment:
These are configuration steps and go in getting started. You only need to do these once.